emacs/redtt.el

```diff
@@ -25,16 +25,15 @@
 ;;; Commentary:
 
 ;; This is a major mode for editing redtt developments.  The current
-;; editing features include simple syntax highlighting. There is a command to
+;; editing features include simple syntax highlighting.  There is a command to
```

Comment:
Really?!

Reply:
Yeah, the elisp package linter is fascistic!

Reply:
I don't remember having to do this for wiki-summary... maybe I just didn't have any comments. Taste it.